Tests/MessagingInApp/EngineWeb/EngineWebProviderStub.swift

```diff
@@ -12,3 +13,17 @@ class EngineWebProviderStub: EngineWebProvider {
         engineWebMock
     }
 }
+
+// In order to test multiple instances of inline Views, you need to create multiple separate EngineWebInstance instances for each View instance.
+// This is a new stub that allows this.
+// A follow-up refactor that makes this stub the default would be a good chance to the test suite.
```

Comment:
I am planning on adding another PR to this stack with this refactor. 

Feel free to not approve this PR until that PR is added to the stack.

Reply:
See https://github.com/customerio/customerio-ios/pull/756/files#r1661310325 to understand why this new stub was created.